README.md

```diff
@@ -13,72 +13,101 @@ To get business, simply:
 $ gem install business
 ```
 
+## Important: 2.0.0 breaking changes
```

Comment:
Could we add a perm link to the `v1.18.0` branch as a courtesy to those who wouldn't want to upgrade?

Reply:
wouldn't they just pin the version in their `Gemfile` to v.1.18.0?

Reply:
True, but theres also a courtesy part where you link to the branch on the README as the documentation would be different.